source/reference/supported-server-version.txt

```diff
@@ -0,0 +1,71 @@
+.. _c2c-server-version-compatibility:
+
+================================================
+MongoDB Server Version Compatibility and Support
+================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Before you run {+c2c-product-name+}, consider the following MongoDB server 
+version limitations and requirements: 
+
+- .. include:: /includes/fact-minimum-server-version-support.rst
+
+- ``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
+
+- .. include:: /includes/fact-minimum-fcv.rst 
+
+Synchronize Data Between Clusters Running Older MongoDB Server Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/fact-minimum-versions.rst
+
+
+Synchronize Data Between Clusters with Different MongoDB Server Major Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/cross-version-sync.rst
+
+``mongosync`` supports syncs between the following source and
+destination MongoDB server versions. The table shows the source MongoDB
+server versions in the left column and destination MongoDB server
+versions in the first row. 
+
+.. list-table:: 
+   :widths: 20 20 20 20 20
+
+   * - 
+     - **4.4** Destination
+     - **5.0** Destination
+     - **6.0** Destination
+     - **7.0** Destination
+
+   * - **4.4** Source
```

Comment:
This table should be changed. Pre-6.0 server versions aren't supported without PS support. So, I think the table should start from 6.0 and go to 7.0.

Reply:
Also, because we're adding that table, I think we should change the one of the paragraphs before from:

"Starting in 1.7.0, mongosync can perform a cross-version migration from a lower major version source cluster to a higher major version destination cluster. You can migrate up to two major versions ahead. For example, you can synchronize a cluster running MongoDB 6.0 with a cluster running 7.0."

TO 

"Starting in version 1.7, mongosync can perform certain cross-version migrations from a lower major version source cluster to a higher major version destination cluster."

I think this helps make the information less redundant.